js/app.js

```diff
@@ -0,0 +1,46 @@
+import credentials from './credentials.js'
+
+
+document.querySelector('form').addEventListener('submit', function(e) {
+    e.preventDefault();
+    let email = document.querySelector('input#search').value
+    console.log(email)
+    var studentInfo = credentials.studentsInfo.filter(stu => {
+        return stu.email = email
```

Comment:
as I see `js/credentials.js`, is it wrong field name? should be 'stu.student_email' instead
```suggestion
        return stu.student_email === email
```